plugins/dominant-color-images/load.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Plugin Name: Dominant Color Images
+ * Plugin URI: https://github.com/WordPress/performance/tree/trunk/plugins/images/dominant-color-images
+ * Description: Adds support to store the dominant color of newly uploaded images and create a placeholder background of that color.
+ * Requires at least: 6.3
+ * Requires PHP: 7.0
+ * Version: 1.0.1
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain: dominant-color-images
+ *
+ * @package dominant-color-images
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+// Define required constants.
+define( 'DOMINANT_COLOR_IMAGES_VERSION', '1.0.1' );
+
```

Comment:
Shouldn't the plugin take precedence over the module if it's published as a plugin?

Reply:
Yeah, that makes sense to me. Old PL versions with that `if` intact will short circuit when the standalone plugins is active.

Reply:
It's safer to keep the `defined( ... )` constant check above the constant to prevent double loading. Yes, this probably shouldn't happen, but IMO better safe than sorry, and the other standalone plugins have that condition too.

If we decide to remove them, let's remove them everywhere rather than making partial decisions.